sisl/viz/__init__.py

```diff
@@ -0,0 +1,24 @@
+"""
+====================================
+Visualization utilities (:mod:`sisl.viz`)
+====================================
+
+.. module:: sisl.viz
+   :noindex:
+
+Classes that are used for visualizing simulations results
+
+Range routines
+==============
+
+Miscellaneous routines
+======================
+
+"""
+
```

Comment:
What happens if the users don't have the modules?
We need to fallback to have the installation without the gui stuff. :)

Reply:
I don't know how to do this or where this sould be done :)